CQUI.modinfo

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.1">
+<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.2">
   <!-- ==============================  Basic Mod Information ============================== -->
   <Properties>
     <AffectsSavedGames>0</AffectsSavedGames>
     <Name>Community Quick User Interface</Name>
     <Teaser>CQUI</Teaser>
     <Description>The Community Quick User Interface MOD, CivFanatics Version: Updated May 2020.</Description>
-    <CompatibleVersions>2.0,2.01</CompatibleVersions>
+    <CompatibleVersions>2.0,2.01,2.1</CompatibleVersions>
```

Comment:
I think this line is for indicating compatibility with different versions of Civilization 6. I tried building one of the sample projects in ModBuddy and the project properties have options for indicating compatibility with the two expansions. The line should be
```suggestion
    <CompatibleVersions>1.2,2.0</CompatibleVersions>
```
1.2 is for Rise and Fall - Spring Update
2.0 is for Gathering Storm

Reply:
I'll fix this now